LanguageFeatures/Augmentation-libraries/augmented_expression_A11_t01.dart

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion The exact result of an `augmented` expression depends on what is
+/// being augmented, but it generally follows the same rules as any normal
+/// identifier:
+/// ...
+/// - Augmenting non-redirecting generative constructors: Unlike other
+///   functions, `augmented` has no special meaning in non-redirecting
+///   generative constructors. It is still a reserved word inside the body of
+///   these constructors, since they are within the scope of an augmenting
+///   declaration.
+///
+/// @description Checks that it is a compile-time error to use an `augmented` in
+/// the body of augmenting non-redirecting generative constructor.
```

Comment:
I think this fix will be applicable in about 9 other locations, too:
```suggestion
/// the body of an augmenting non-redirecting generative constructor.
```

Reply:
Thanks!